src/backend/InvenTree/InvenTree/admin.py

```
@@ -1,5 +1,7 @@
 """Admin classes."""
 
+from typing import List
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)